### PR TITLE
{vis}[GCCcore/8.3.0] GL2PS v1.4.0

### PR DESCRIPTION
**Pull request by furstj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/889bdc35a39fe7e48b793f265f363f09 for a full test report.

File: easybuild/easyconfigs/g/GL2PS/GL2PS-1.4.0-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    ('X11', '20190717'),
+    ('Mesa', '19.1.7'),
+    ('libGLU', '9.0.1'),
+    ('freeglut', '3.2.1'),
+    ('libpng', '1.6.37'),
+    ('zlib', '1.2.11'),
+]
+
+sanity_check_paths = {
+    'files': ['include/gl2ps.h', 'lib/libgl2ps.so'],
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/g/GL2PS/GL2PS-1.4.0-GCCcore-8.3.0.eb]: `'lib/libgl2ps.%s' % SHLIB_EXT` would be more portable

Comment (furstj) [reply easybuild/easyconfigs/g/GL2PS/GL2PS-1.4.0-GCCcore-8.3.0.eb]: Changed in last commit

**Comment by migueldiascosta** [comment]
Going in, thanks @furstj!

**Pull request merged**